api/app/Mappers/UserDTOMapper.php

```diff
@@ -5,24 +5,46 @@
 
 use App\Models\User;
 use App\DTOs\UserDTO;
+use Carbon\Carbon;
 use \DateTime;
 
 class UserDTOMapper
 {
-    public static function mapUser(User $user) : UserDTO
+    public static function mapModelToUserDTO(User $user) : UserDTO
```

Comment:
Which model? "ToUserDTO" can be omitted because it is already included into the method signature

Reply:
ok